docs/source/contribute.rst

```diff
@@ -19,8 +23,15 @@ Prof. Dr. Rens van de Schoot <https://www.rensvandeschoot.com/contact/>`_ or sen
 Contribute
 ~~~~~~~~~~
 
-If you discover issues please let us know via `Github
-<https://github.com/asreview/asreview/issues/new/choose>`_. If you have ideas
-to solve your own or other issues, it is highly appreciate to contribute to
-the `development <https://github.com/asreview/asreview/blob/master/CONTRIBUTING.md>`_.
+How do you go from user to contributor? There are many ways to join in, and it might be less complicated than you expect. In a `blogpost <https://asreview.nl/blog/open-source-and-research/>`_ we list some easy examples for first-time contributors, for example sharing your experiences or answering users questions on the `Dicsussion platform <https://github.com/asreview/asreview/discussions>`_. 
```

Comment:
```suggestion
How do you go from user to contributor? There are many ways to join in, and it might be less complicated than you expect. In a `blogpost <https://asreview.nl/blog/open-source-and-research/>`_, we list some easy examples for first-time contributors, for example sharing your experiences or answering user questions on the `Discussion platform <https://github.com/asreview/asreview/discussions>`_. 
```